stories/SQFormReadOnlyField.stories.tsx

```diff
@@ -46,3 +56,5 @@ WithInitialValue.args = {
 WithInitialValue.parameters = {
   controls: {exclude: 'initialValue'},
 };
+
+export default meta;
```

Comment:
There should be a newline at the end of the file

Reply:
:shakes fist at vscode: